data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
There is also the problem of equipping the item, lets say you ahve a shield in your hand and you buy another shield in the store, if you move the shield in the store inbox it will replace with your equipped one and that shouldn't happen.